server/models/ImageUrl.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require("mongoose");
+
+const imgUrlSchema = new mongoose.Schema({
+  filePath: {
```

Comment:
I think it would be more appropriate if we add this field to the Profile model instead :)